src/vm/i386/excepx86.cpp

```diff
@@ -3769,10 +3783,27 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifndef DACCESS_COMPILE
+#ifndef WIN64EXCEPTIONS
 #ifdef FEATURE_PAL
+EXTERN_C
+_Unwind_Reason_Code
+UnhandledExceptionHandlerUnix(
```

Comment:
Ah, I got it. I just want to locate both ``UnhandledExceptionHandlerUnix`` and ``DispatchManagedException`` in the same place as both functions are implemented in ``exceptionhanding.cpp``, and will be removed once ``WIN64EXCEPTIONS`` is enabled.